habitat/datasets/rearrange/samplers.py

```diff
@@ -383,62 +440,65 @@ def __init__(
             x.creation_attributes.handle for x in self.object_instance_set
         ]
         super().__init__(
-            object_set, receptacle_sets, num_targets, orientation_sample
+            object_set,
+            receptacle_sets,
+            num_targets,
+            orientation_sample,
+            sample_region_ratio,
+            nav_to_min_distance,
         )
 
     def sample(
         self,
         sim: habitat_sim.Simulator,
         snap_down: bool = False,
         vdb: Optional[DebugVisualizer] = None,
-    ) -> Optional[Dict[str, habitat_sim.physics.ManagedRigidObject]]:
+        target_receptacles=None,
+        goal_receptacles=None,
+        object_to_containing_receptacle=None,
```

Comment:
Are these accurate? May want to document the details here in the docstring also.
```suggestion
        target_receptacles: Optional[List[Receptacle]]=None,
        goal_receptacles: Optional[List[Receptacle]]=None,
        object_to_containing_receptacle: Optional[Dict[str, Receptacle]]=None,
```